code/modules/power/generator.dm

```diff
@@ -99,6 +108,7 @@
 	lastgen -= power_output
 	..()
 
+/* todo: destroy
```

Comment:
destroy

Reply:
@martinlyra

Reply:
Right so, this was only comented out when I needed to glance at how the original HTML ui worked, you dont need this anymore

Reply:
destroyed:tm: